Changes.CIAO_scripts

```diff
@@ -6,6 +6,9 @@
     Updated detsubsys parameter to recognize the new ACIS frame store 
     shadow bit (17).
 
+    Updated to support HRC-I+LETG LSFPARM files if they are available
+    in the CALDB.
```

Comment:
Is there a CALDB version where they were added (so we can say something like "if you have CALDB vblah or later"), or is it more complicated taht that?

Reply:
There will be (assuming Dale's release happens at same time as contrib).